.github/workflows/generate_ebooks.yml

```diff
@@ -0,0 +1,77 @@
+######################################
+## Custom Web Almanac GitHub action ##
+######################################
+#
+# Generate the ebooks when this GitHub Action is run manually
+# This should be run everytime before release
+#
+name: Generate Ebooks
+
+env:
+  # Update periodically from https://www.princexml.com/latest/
+  PRINCE_PACKAGE: 'prince_20200728-1_ubuntu20.04_amd64.deb'
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    # Prince needs ubuntu 20 and ubuntu-latest is only on 18
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+    - name: Setup Node.js for use with actions
+      uses: actions/setup-node@v1.1.0
+      with:
+        version:  12.x
+    # Install Python 3.7 as used by Google Cloud Platform
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install Python requirements
+      run: |
+        echo "Installing Python requirements"
```

Comment:
Name of the step makes it obvious, no need to echo it explicitly.

```suggestion
```

Reply:
GitHub actions have this annoying thing where they list all the run commands under the first command instead of under the name:

![image](https://user-images.githubusercontent.com/10931297/88868702-c8e14800-d208-11ea-995d-fe8b9888f4c2.png)

When expanded you can see the real commands:

![image](https://user-images.githubusercontent.com/10931297/88868736-e0b8cc00-d208-11ea-91d1-9a70b53af9a2.png)

The Echo was an attempt to at least make that grouping name more meaingful:

![image](https://user-images.githubusercontent.com/10931297/88868788-034ae500-d209-11ea-9e9c-df119bde2270.png)

![image](https://user-images.githubusercontent.com/10931297/88868798-0b0a8980-d209-11ea-9fbe-af73c172f0bf.png)

Any ideas for a better way to solve this?

Reply:
I usually set a secret named `ACTIONS_STEP_DEBUG` and set it to `true` so that I can see more detailed output when I need it. Otherwise, in a normal code run knowing that all the steps passed is more than enough.

Reply:
OK removed all the `echo`s.